index.rst

```diff
@@ -6,24 +6,474 @@
+  pip install --user pycodestyle_magic
+
+It is known that the most up-to-date version of flake8 has some issues.
+If errors are encountered such as "AttributeError: '_io.StringIO' object has no attribute 'buffer'", 
+force-downgrade ``flake8`` from vesion 4.0.1 to 3.9.2 with ``pip install flake8=3.9.2``.
+
+
+The flake8 config file
+^^^^^^^^^^^^^^^^^^^^^^
+
+Create a configuration file for ``flake8``.
+
+These instructions use emacs but it doesn’t matter so long as the end result is correctly-named file with the right contents. 
+For example, from the command line in your home directory, execute:
+
+::
+
+  touch .config/flake8
+  emacs .config/flake8
+
+
+Then copy-paste the following into the opened config file:
+
+::
+
+  [flake8]
+  max-line-length = 99
+  ignore = E133, E226, E228, E266, N802, N803, N806, N812, N813, N815, N816, W503
+
+Use ``x-s`` then ``x-c`` to save and exit emacs.
+
+
+While developing a notebook
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While developing a notebook, have the following 'magic' commands as the first code cell:
+
+::
+
+  %load_ext pycodestyle_magic
+  %flake8_on
+  import logging
+  logging.getLogger("flake8").setLevel(logging.FATAL)
+
+Whenever you execute a cell, it will use ``flake8`` to check for adherence to the ``PEP8`` coding style guide, and report violations.
+Fix them as you go.
+Once you're done with the entire notebook you can remove that cell with the magic commands. 
+
+
+When the notebook is complete
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When the notebook is complete, from the command line in the notebook's directory execute:
+
+::
+
+  flake8-nb notebook_name.ipynb
+
+
+This will give you a final check of any violations with ``PEP8``.
+This will catch things that can be missed line-by-line, such as packages that are imported but never used.  
+
+
+Git branch, merge, and review policy for ``tutorial-notebooks``
+===============================================================
+
+The following applies when creating or updating notebooks in the ``tutorial-notebooks`` repository of the ``rubin-dp0`` GitHub Org.
+
+Branch
+------
+
+Develop new notebooks, or update existing ones, in a new branch.
+This branch should be named for the corresponding Jira ticket (e.g., "tickets/PREOPS-12345").
+The new branch should be made from ``main``, *not* from ``prod``.
+
+Typically, only update one notebook per ticket branch, unless the ticket is to make similar updates to all notebooks
+(e.g., when bumping the RSP's recommended image).
+
+Update the README file when appropriate.
+
+Commit and push
+---------------
+
+Always restart the kernel and clear all outputs before saving, committing, and pushing changes.
+
+Pull request
+------------
+
+When the notebook is complete open a pull request to merge the ticket branch into the ``main`` branch (*not* to ``prod``). 
+
+Review
+------
+
+Contact one or more Rubin staff members (it need not be a Community Engagement team member) with the appropriate expertise
```

Comment:
Where could people ask for reviewers if they don't know anyone a priori? Perhaps in "Community"?

Reply:
Added link and instructions to post on community.lsst.org